### PR TITLE
[Doc Update] [OE Schema: Remove one required field] [3718295]

### PR DESCRIPTION
**Pull request by erikadoyle**
Request from Outlook Extensions feature team to make actionId property optional (remove required checkmark).

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/81b97b73fd0a1c123380c4cc679608da12d90389/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 132 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1a08c68-33f3-44f0-aa9a-2f9ff15360cb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
 |`tabs.customMobileRibbonGroups.controls.id` | String | 250 characters | ✔️ | Specifies the ID of the control such as "msgReadFunctionButton".|
 |`tabs.customMobileRibbonGroups.controls.type` | String enum |  | ✔️ | Specifies the type of control. Currently only "MobileButton" is supported.|
 |`tabs.customMobileRibbonGroups.controls.label` | String | 32 characters | ✔️ | Specifies the label on the control.|
-|`tabs.customMobileRibbonGroups.controls.actionId` | String | 64 characters |✔️ | Specifies the ID of the action that is taken when a user selects the control. The `actionId` must match the `runtime.actions.id` property of an action in the `runtimes` object.|
+|`tabs.customMobileRibbonGroups.controls.actionId` | String | 64 characters | | Specifies the ID of the action that is taken when a user selects the control. The `actionId` must match the `runtime.actions.id` property of an action in the `runtimes` object.|
```
Comment (v-ypalikila) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:1066]: The property `actionId` in the  `extensionCommonCustomControlMenuItem` is marked as required in the [Dev Preview schema](https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json).

Is the update coming as part of a new schema version? Request you to please check and confirm.

image

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 2]
```diff
@@ -1040,7 +1040,7 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
 |`tabs.groups.controls.items.supertip`| | |✔️| Configures a supertip for the custom item. A supertip is a UI feature that displays a brief box of help information about a control when the cursor hovers over it. The box may contain multiple lines of text. |
 |`tabs.groups.controls.items.supertip.title`| String | 64 characters | ✔️ | Specifies the title text of the supertip.|
 |`tabs.groups.controls.items.supertip.description`| String | 128 characters | ✔️ | Specifies the description of the supertip.|
-|`tabs.groups.controls.items.actionId`| String | 64 characters | ✔️ | Specifies the ID of the action that is taken when a user selects the control or menu item. The `actionId` must match with `runtime.actions.id`. |
+|`tabs.groups.controls.items.actionId`| String | 64 characters | | Specifies the ID of the action that is taken when a user selects the control or menu item. The `actionId` must match with `runtime.actions.id`. |
```
Comment (v-ypalikila) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:1043]: The property `actionId` in the  `extensionCommonCustomControlMenuItem` is marked as required in the [Dev Preview schema](https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json).

Is the update coming as part of a new schema version? Request you to please check and confirm.

image

**Review by v-ypalikila — CHANGES_REQUESTED** [review]
Request you to look into the comments and provide your inputs.

**Comment by v-ypalikila** [comment]
@erikadoyle , A gentle reminder to look into the comments and share your inputs.

**Comment by erikadoyle** [comment]
Thank you @v-ypalikila, yes--these changes are pending in devPreview pipeline. I drafted the PR just so we're ready once manifest shiproom is ready to release.

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 2 above)
Comment (erikadoyle) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md:1043]: Confirmed with final schema update

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 1 above)
Comment (erikadoyle) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md:1066]: Confirmed with final schema update

**Review by Rick-Kirkham — APPROVED** [review]

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c6895cc4fb6cec4d94470b100ad8f58b395faa98/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 131 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a3114e5-9d71-407a-b8af-a369856b0e0b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/35f5ed435053c28507132beada26464d8a1d9009/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 84 | 126 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c17b68fa-32b7-4009-9d66-83a4213b7f7d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**